src/adapter/activateDaffodilDebug.ts

```diff
@@ -103,6 +103,18 @@ function normalizePath(path: string): string {
   return path
 }
 
+/**
+ * Configures the file path string of an intended TDML save file if the
+ * extension is malformed.
+ * @param pathStr
+ * @returns file path string with a valid TDML file extension.
+ */
+function validateTDMLFilePath(pathStr: string): string {
+  return !pathStr.match(/.*tdml[.]xml|.*tdml$/gm)
```

Comment:
Probably ought to make sure the file name ends with `.tdml` not just `tdml` (e.g., foo.tdml vs footdml).